src/main/scripts/cdp-listend

```diff
@@ -239,6 +247,7 @@ cdp-listend [OPTIONS]
   --fetch_smear=SECONDS   fetch run smearing time
   --nch_smear=SECONDS     nch run smearing time\n";
   --facility              syslog facility
+  --hostname=fqdn         hostname/IP to listen on (use 0.0.0.0 to listen on all IPs)
```

Comment:
@ned21 is it sufficient if i remove the 0.0.0.0 from the help text? 

the old code would only listen on whatever hostname is set to, external/public or not. only localhost is not allowed (altough that is also not really valid anymore, people could listen on localhost and have some routing/kernel rules in places that pick up the packets from other interfaces.)

wrt security, in order to solve #13 properly, i'll add optional support for encrypting messages, similar to what ccm-fetch uses with TRUST. that way you could listen on 0.0.0.0; but that's for later.